From 33df3f5338257324af0eac055d5c4e90f9da57f8 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Sat, 17 May 2014 12:28:23 +0200
Subject: [PATCH] tests: remove target for unneeded tests which let the build
 fail

Gives the following error with automake 1.12:

autoreconf: running: automake --foreign --add-missing --copy --force-missing
automake: warnings are treated as errors
src/tests/Makefile.am:62: warning: user target 'buildtest-TESTS' defined here ...
/home/morphis/work/wop/webos-ports/tmp-eglibc/sysroots/x86_64-linux/usr/share/automake-1.12/am/check.am:
... overrides Automake target 'buildtest-TESTS' defined here
autoreconf: automake failed with exit status: 1
ERROR: autoreconf execution failed.
WARNING: exit code 1 from a shell command.
---
 src/tests/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/tests/Makefile.am b/src/tests/Makefile.am
index 641e207..2501b93 100644
--- a/src/tests/Makefile.am
+++ b/src/tests/Makefile.am
@@ -59,8 +59,6 @@ TESTS = lxc-test-containertests lxc-test-locktests \
         lxc-test-snapshot lxc-test-concurrent lxc-test-may-control \
         lxc-test-reboot lxc-test-list lxc-test-attach lxc-test-device-add-remove
 
-buildtest-TESTS: $(TESTS)
-
 install-ptest:
 	install -d $(TEST_DIR)
 	install -D ../lxc/liblxc.so $(TEST_DIR)/../lxc/liblxc.so
-- 
1.9.1

